src/components/checkboxes/_checkbox.scss

```diff
@@ -36,11 +36,11 @@ $checkbox-label-vertical-padding: 0.75rem;
   &__label {
     cursor: pointer;
     display: block;
-    padding: $checkbox-padding 1rem $checkbox-padding ($checkbox-padding * 2 + $checkbox-input-width);
+    padding: $checkbox-padding 18px $checkbox-padding ($checkbox-padding * 2 + $checkbox-input-width);
     width: 100%;
 
     &--with-description {
-      padding: 0 1rem $checkbox-padding 0;
+      padding: 0 18px $checkbox-padding 0;
```

Comment:
Think this can be left as `1em` because it's dealing with vertical typography height (em height)